integration-cli/docker_api_containers_test.go

```diff
@@ -552,7 +552,7 @@ func (s *DockerSuite) TestContainerApiCommit(c *check.C) {
 	if err != nil {
 		c.Fatal(err)
 	}
-	if cmd != "{[/bin/sh -c touch /test]}" {
```

Comment:
Not completely sure. But my guess is `{}` was to handle the case when the template was applied again in case `err != nil`. Once that code was removed, this `{}` was not needed either.